src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -177,21 +192,21 @@ export const DepartureDetailsScreenComponent = ({
         }}
         parallaxContent={(focusRef) => (
           <View style={styles.parallaxContent}>
-            <View style={styles.headerTitle} ref={focusRef} accessible={true}>
-              {mode && (
-                <TransportationIconBox
-                  mode={mode}
-                  subMode={subMode}
-                  style={styles.headerTitleIcon}
-                />
-              )}
+            <View
+              style={styles.headerTitle}
+              ref={focusRef}
+              accessible={true}
+              accessibilityLabel={a11yLabel}
+            >
+              {serviceJourney && <LineChip serviceJourney={serviceJourney} />}
               <ThemeText
-                type="heading--medium"
+                type="heading__title"
                 color={themeColor}
-                style={{flexShrink: 1}}
+                style={{flex: 1}}
               >
                 {title ?? t(DepartureDetailsTexts.header.notFound)}
               </ThemeText>
+              {fromQuay && <DepartureTime departure={fromQuay} />}
```

Comment:
@rosvik or @gorandalum how do I get a EstimatedCall from EstimatedCallWithMetadata? fromQuay is EstimatedCallWithMetadata

Reply:
Yes, I did that but the function I want to use is getLineAndTimeA11yLabel and internally it access objects that are not part of EstimatedCallWithQuayFragment

Reply:
It also subcalls another functions with other properties and I ended up adding | EstimatedCallWithQuayFragment to 5 or more functions but there was one that missed data.

Reply:
This conversation was discussed internally at the office. We decided to use a different approach since EstimatedCall and EstimatedCallWithMetadata are two other models.